server/mergin/sync/private_api.yaml

```diff
@@ -445,6 +477,29 @@ components:
           requested_by:
             type: string
             example: jane.doe
+    ProjectRequestAccessListItem:
```

Comment:
Just suggestion:

- rename to ProjectAccessRequest (or ProjectAccessRequestItem)
- use it in schema ProjectRequestAccessList (maybe also rename in right order)

```
    # Array of Pets
    type: array
    items:
      $ref: '#/components/schemas/Pet'
 ```